src/practice/js/index.js

```diff
@@ -5,4 +5,40 @@ import { createElements, createErrorElement } from "./createElement";
 window.addEventListener('DOMContentLoaded', () => {
   // ここに処理を書いていく
 
+  // 1,全ポケモンを取得
+  $axios('https://pokeapi.co/api/v2/pokemon/?limit=151').then(response => {
+    const results = response.data.results; // ポケモン一覧
```

Comment:
[nits]
結果は一つなため単数系でお願いします。
```suggestion
    const result = response.data.result; // ポケモン一覧
```